resources/js/lib/index.ts

```diff
@@ -3,18 +3,18 @@ import Axios from 'axios';
 export const axios = Axios.create();
 
 if (typeof window !== 'undefined') {
-  const token = document.head.querySelector('meta[name="csrf-token"]') as HTMLMetaElement | null;
-  axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
-  axios.defaults.headers.common.Accept = 'application/json';
-  axios.defaults.headers.patch['Content-Type'] = 'application/json';
-  axios.defaults.headers.post['Content-Type'] = 'application/json';
+    const token = document.head.querySelector('meta[name="csrf-token"]') as HTMLMetaElement | null;
+    axios.defaults.headers.common['X-Requested-With'] = 'XMLHttpRequest';
+    axios.defaults.headers.common.Accept = 'application/json';
+    axios.defaults.headers.patch['Content-Type'] = 'application/json';
+    axios.defaults.headers.post['Content-Type'] = 'application/json';
 
-  if (token) {
-    axios.defaults.headers.common['X-CSRF-TOKEN'] = token.content;
-  }
+    if (token) {
+        axios.defaults.headers.common['X-CSRF-TOKEN'] = token.content;
+    }
```

Comment:
Suggestion: Fallback if no csrf-token is present, use cookie-based CSRF

```suggestion
    } else {
        axios.defaults.withCredentials = true;
    }
```